src/components/pages/career-details/career-details.tsx

```diff
@@ -53,7 +58,7 @@ export const CareerDetails = ({
 
       <CareerForm
         company_id="41230"
-        recruiting_channel_id="329206"
+        recruiting_channel_id={personioChannel ?? '329206'}
```

Comment:
please make the number a speaking constant like `DEFAULT_CHANNEL_ID` and reference it. It's much more readable than this.

Reply:
looks better, I fixed it 👍